extensions/wikia/AffiliateService/js/ext.AffiliateService.js

```diff
@@ -1,3 +1,39 @@
+function trackingPairsToObject(unit) {
+	var tracking = {
+		algo: '',
+		method: '',
+		version: '',
+		recommendation_level: '',
+	};
+
+	if (unit.tracking && unit.tracking.forEach && typeof unit.tracking.forEach === 'function') {
+		unit.tracking.forEach((kv) => {
+			tracking[`${kv.key}`] = kv.val;
```

Comment:
Please don't use backticks in ES5; also those are not needed here

Reply:
https://github.com/Wikia/app/pull/17805/commits/44f4b75fb1bedeb2fa531bb72e1fea85efee93ec